core/src/main/resources/jenkins/model/MasterBuildConfiguration/config.jelly

```diff
@@ -0,0 +1,30 @@
+<!--
+The MIT License
+Copyright (c) 2025, Stefan Spieker
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry title="${%# of executors}" field="numExecutors">
+        <f:number min="0" clazz="non-negative-number-required"/>
```

Comment:
```suggestion
        <f:number min="0" clazz="non-negative-number-required" step="1"/>
```

Technically unnecessary though, as `1` is the default `step` for a `number` text field.

I recommend intentional omissions in transformation PRs like this to be line-commented by the author, then it's clear that it's intentional. Right now it looks like an oversight.